src/client/components/DelayComponent.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default (args = {}) => Element =>
+  class DelayComponent extends React.Component {
+    constructor() {
+      super();
+      this.state = { ready: true };
+    }
+
+    componentWillMount() {
+      const { delay, onRender } = args;
+      const d = parseInt(delay, 10);
+      if (d && d > 0) {
+        this.setState({ ready: false });
+        this.timeout = setTimeout(() => {
```

Comment:
There should be a better solution than adding timeout

Reply:
I'm pretty sure its this thats preserving the auth state - https://github.com/busyorg/busy/blob/ca3d908ad046461cf65263aace805aec92123209/src/server/handlers/createSsrHandler.js#L48

Look at my gif here: https://giphy.com/gifs/3ohs4xmAKooN3hwUtG/fullscreen

I was thinking of maybe preserving the state for everything else except `auth` for SSR

Reply:
@Sekhmet might have an idea, let me know your thoughts